### ISSUE TITLE
[Typescript] Unexpected exception in dev-machine console

### ISSUE THREAD

**Issue by SkorikSergey** [eclipse-che/che#12017]
### Reproduction Steps
* Create a workspace based on **Node** stack with enabled **TypeScript** language server
* Create typescript file and add content:
```
function greeter(person) {
    return "Hello, " + person;
}
```
* Type ```greeter(```, wait for signatures proposal and type any symbol.

**Actual result:** There is ```Exception occurred in op
java.util.concurrent.ExecutionException: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Request textDocument/signatureHelp failed with message: No content available.``` exception in **dev-machine** console.

**Che version**: 6.14.0
**OS and version:** Fedora 28

**Additional information:**
* Problem can be reliably reproduced, doesn't happen randomly: [Yes]
* See the attachment:

**Screenshot:**
![selection_162](https://user-images.githubusercontent.com/7760565/48853096-7145d480-edb7-11e8-952a-94a76f283831.png)

**Screencast:**
![screencast-172 19 20 234-8080-2018 11 21-17-53-07](https://user-images.githubusercontent.com/7760565/48853074-68550300-edb7-11e8-9aed-7a54504606d6.gif)

**Comment by SkorikSergey** [comment]
This issue started to reproduce from CHE 6.14.0.

**Comment by dkuleshov** [comment]
Okay, I [see](https://github.com/eclipse/che/compare/6.13.0...master) that #10699 was merged in 6.13.0 so it is not related, this is strange.

**Comment by SkorikSergey** [comment]
Not relevant anymore. Closing.
